projects/sandbox/src/app/wizard/wizard.component.html

```diff
@@ -74,15 +74,17 @@
       <div *ngIf="currentStepIndex === 1" class="form-group">
         <label for="country" i18n="country label">Country</label>
         <select id="country" formControlName="country">
-          <option *ngFor="let option of countryList" [ngValue]="option">{{ option }}</option>
+          <option *ngFor="let option of countryList" [ngValue]="option.xmlValue">{{
+            option.xmlValue
+          }}</option>
         </select>
       </div>
 
       <div *ngIf="currentStepIndex === 1" class="form-group">
         <label for="language" i18n="language label">Language</label>
         <select id="language" formControlName="language">
-          <option *ngFor="let option of languageList" [ngValue]="option.code">{{
-            option.name
+          <option *ngFor="let option of languageList" [ngValue]="option.xmlValue">{{
+            option.xmlValue
```

Comment:
I think for languages the value that's sent back (i.e. the option's value attribute) should be equal to the language's or country's name property (and not the xmlValue property). One is for display, the other is for internal use. They just happen to be the same (apart from whether they are lower case or upper case) but this may change.